example/example.g.dart

```diff
@@ -26,20 +26,22 @@ abstract class _$PersonSerializerMixin {
   DateTime get lastOrder;
   List<Order> get orders;
   Map<String, dynamic> toJson() {
-    var $map = <String, dynamic>{};
-    void $writeNotNull(String key, dynamic value) {
+    var val = <String, dynamic>{
+      'firstName': firstName,
+    };
+
+    void writeNotNull(String key, dynamic value) {
```

Comment:
taking a look at this - I might skip the extra method and inline the conditionals through the rest. It's noisy but shouldn't matter since it's generated code